src/omlt/neuralnet/activations/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 ACTIVATION_FUNCTION_MAP = {
     "linear": linear_activation_function,
-#    "relu": bigm_relu_activation,
+#    "relu": relu_max_activation
     "sigmoid": sigmoid_activation_function,
     "softplus": softplus_activation_function,
     "tanh": tanh_activation_function
```

Comment:
Does `ACTIVATION_FUNCTION_MAP` need to be here? We import it as `_DEFAULT_ACTIVATION_FUNCTIONS` into `nn_formulation.py`, but then we define `_DEFAULT_ACTIVATION_CONSTRAINTS` separately in that file. Maybe these are attributes we want to expose to the user? If yes, should we put the constraint map here too?